.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
```

Comment:
We also have a [`Build & Test` workflow](https://github.com/1Password/terraform-provider-onepassword/blob/c3fe693801bae9140fb46af1399ff780f59b2912/.github/workflows/build.yml#L1).

I believe naming this workflow `Test` may create the expectations that we run the go test suite in this workflow.

How about naming this workflow `Verify Docs` instead?